src/main/java/org/openstreetmap/atlas/geography/atlas/change/ChangeBuilder.java

```diff
@@ -48,6 +52,42 @@ public synchronized ChangeBuilder add(final FeatureChange featureChange)
         return this;
     }
 
+    /**
+     * Iteratively {@link #add(FeatureChange)} all the FeatureChanges.
+     *
+     * @param featureChanges
+     *            - The featureChanges to add.
+     * @return ChangeBuilder - returns itself to allow fluency in calls.
+     */
+    public synchronized ChangeBuilder addAll(final Stream<FeatureChange> featureChanges)
+    {
+        featureChanges.map(this::add).reduce((first, second) -> second);
```

Comment:
Why not use forEach here to avoid the reduce?

Reply:
Good point.